java/src/jmri/jmrix/loconet/LnSensor.java

```diff
@@ -47,14 +47,13 @@ private void init(String systemName, String prefix) {
 
     /**
      * Request an update on status by sending a LocoNet message.
+     * The only known way to do this from LocoNet is to request the
```

Comment:
Be aware that some devices do not respond to the referenced "interrogate" messages.  And on a very busy LocoNet, "collisions happen", which can prevent some sensor status messages from getting thru.  In other words, a user may not be able to reliably get status for all of the sensors.

Reply:
I'm not a LocoNet user @devel-bobm , this was code comments moved to Javadoc.
Would your comments on this fit better in code comments / Javadoc / user html ?